src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public Bulldozer(String machine) {
+        super(machine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(machine + " started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(machine + " stopped working.");

```

Comment:
Similarly, specify that it is a `Bulldozer` that stopped working in the message for consistency and informativeness.